com.unity.render-pipelines.universal/ShaderLibrary/Shadows.hlsl

```diff
@@ -211,31 +207,57 @@ half SampleScreenSpaceShadowmap(float4 shadowCoord)
 
 real SampleShadowmapFiltered(TEXTURE2D_SHADOW_PARAM(ShadowMap, sampler_ShadowMap), float4 shadowCoord, ShadowSamplingData samplingData)
 {
-    real attenuation;
+    real attenuation = 1.0f;
```

Comment:
I would assume this gives a warning when compiling the shader. 
Probably need to do `real attenuation = real(1.0);`

Reply:
fixed